### ISSUE TITLE
rocksdb wal文件可以删除吗

### ISSUE THREAD

**Issue by songlf1002** [apache/incubator-hugegraph#2369]
### Problem Type (问题类型)

None

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/apache/hugegraph/issues) 与 [FAQ](https://hugegraph.apache.org/docs/guides/faq/) 中没有相同 / 重复问题 (I have confirmed and searched that there are no similar problems in the historical issue and documents)

### Environment (环境信息)

- Server Version: 0.12.0 (Apache Release Version)
- Backend: RocksDB 1 nodes, SSD 
- OS: xx CPUs, xx G RAM, Ubuntu 2x.x / CentOS 7.x 
- Data Size:  110000vertices, 1300000 edges 

### Your Question (问题描述)

rocksdb存储，使用hugegraph-loader每日全量加载数据，wal文件一直变大，请问wal文件可以删除吗

**Comment by imbajin** [comment]
Yes, u could **delete wal file directly** if u **understand** the **risk** of it (WAL: _Write Ahead Log_, u could search the meaning first)

However, from your case, u should better **reduce** the `WAL TTL`(Auto Delete) time or reduce the max size of `WAL` file, u could search it in `RocksDBOptions`(Normally WAL files are **automatically deleted**(TTL), adjusting the parameters is the best choice)
